sdk/redisenterprise/Microsoft.Azure.Management.RedisEnterpriseCache/src/Microsoft.Azure.Management.RedisEnterprise.csproj

```diff
@@ -11,17 +11,17 @@
     <PackageTags>Microsoft Azure Redis Enterprise cache management;Redis Enterprise Management;</PackageTags>
```

Comment:
I don't think it's right to keep the version here `2.0.0` unchanged. This update definitely contains some breaking changes, so you must bump the major version.